cmd/osctl/cmd/cluster.go

```diff
@@ -335,6 +335,7 @@ func init() {
 	clusterUpCmd.Flags().StringVar(&image, "image", "docker.io/autonomy/talos:"+version.Tag, "the image to use")
 	clusterUpCmd.Flags().StringVar(&networkMTU, "mtu", "1500", "MTU of the docker bridge network")
 	clusterUpCmd.Flags().IntVar(&workers, "workers", 1, "the number of workers to create")
+	clusterUpCmd.Flags().IntVar(&masters, "masters", 3, "the number of masters to create")
```

Comment:
I liked the idea of defaulting to 1. I think HA locally has few benefits outside of our basic integration test. Not a blocker for me though.

Reply:
I was afraid of breaking something else or someone's else automation by changing defaults here.